meta-luv/classes/luv-efi.bbclass

```diff
@@ -172,7 +172,9 @@ python build_efi_cfg() {
        cfgfile.write('default=bits\n')
        cfgfile.write('timeout=0\n')
        cfgfile.write('fallback=0\n')
-
+    else:
+       cfgfile.write('default=luv\n')
+       cfgfile.write('timeout=5\n')
```

Comment:
Wouldn't it be better to simply take the line "cfgfile.write('timeout=0\n') out of the if re.search("(x86_64|i.86)", target): condition?